main.py

```diff
@@ -21,11 +21,11 @@ async def on_ready():
 async def test_image_fun(img_path):
     return
 
-@bot.command(name="test_image", description="Test that the bot can download images and send them back")
-async def test_image(ctx, *args):
+@bot.command(name="echo", description="Echos back the image")
+async def echo(ctx, *args):
     await process_command(ctx, test_image_fun, *args)
 
-@test_image.error
+@echo.error
 async def image_error_handler(ctx, error):
```

Comment:
would do `echo_error_handler` for consistency. 
Maybe we finally move these functions to a different "test" module or something?

Reply:
I don;t think we need it yet, since we only have this one test function currently.